doc/build/doing_build.rst

```diff
@@ -99,24 +99,14 @@ start building Kerberos; see :ref:`do_build`)::
 
     make check
 
-However, there are several prerequisites that must be satisfied first:
-
-* Configure and build Kerberos with Tcl support. Tcl is used to drive
-  the test suite.  This often means passing **-**\ **-with-tcl** to
-  configure to tell it the location of the Tcl configuration
-  script. (See :ref:`options2configure`.)
-* In addition to Tcl, DejaGnu must be available on the system for some
-  of the tests to run.  The test suite will still run the other tests
-  if DejaGnu is not present, but the test coverage will be reduced
-  accordingly.
-* On some operating systems, you have to run ``make install`` before
-  running ``make check``, or the test suite will pick up installed
-  versions of Kerberos libraries rather than the newly built ones.
-  You can install into a prefix that isn't in the system library
-  search path, though. Alternatively, you can configure with
-  **-**\ **-disable-rpath**, which renders the build tree less suitable for
-  installation, but allows testing without interference from
-  previously installed libraries.
+On some operating systems, you have to run ``make install`` before
+running ``make check``, or the test suite will pick up installed
+versions of Kerberos libraries rather than the newly built ones.  You
+can install into a prefix that isn't in the system library search
+path, though.  Alternatively, you can configure with
+**-**\ **-disable-rpath**, which renders the build tree less suitable
```

Comment:
Is this backward?  (To my mind, it doesn't make sense for disabling rpath to make the tree less suitable for installation - files that live on the system should not be using DT_RPATH, right?)

Reply:
If you build with the default prefix (`/usr/local`) or another prefix that isn't in the system default library path, you need rpaths for the binaries to find the libraries.  A distribution building a package for /usr probably does want to use `--disable-rpath`, though.  We could address that somewhere in doing_build.rst, but it's not in scope for this PR (the text here is just being moved out of a bullet list without change) or this section of the file.